cfme/services/myservice/ssui.py

```diff
@@ -230,14 +230,15 @@ def launch_vm_console(self, catalog_item):
 @MiqImplementationContext.external_for(MyService.retire, ViaSSUI)
 def retire(self):
     view = navigate_to(self, 'Retire')
-    view.retire.click()
-    view = self.create_view(MyServicesView)
-    assert wait_for(
-        lambda: view.is_displayed, delay=3, num_sec=300,
-        message="waiting for view to be displayed"
-    )
-    # TODO - remove sleep when BZ 1518954 is fixed
-    time.sleep(10)
+    if self.appliance.version <= "5.9":
```

Comment:
How about for 5.10 ? Services cannot be retired in 5.10 ?

Reply:
Services are getting retired in 5.10 but confirmation pop-up window is not appearing .
so I raised the bug https://bugzilla.redhat.com/show_bug.cgi?id=1653288

Reply:
Then we should mark the test for 5.10 with this BZ

Reply:
We can still retire this vm for now, but we want this stretched use case to be considered for safer side (we don't want customer to click retire by mistake so we have raised issue). However, we have added #to-do there and we will add pop-up thing as well as this thing get fixed in UI, so final thought, it should be not blocked.

Reply:
Okay I thought its being ignored for 5.10 as of now . 
If it's being tested for 5.9 and 5.10 both with a workaround then that is good :+1: